### PR TITLE
[incubator/kubewatch] add kubewatch chart

### PR DESCRIPTION
**Pull request by compleatang**
This PR adds a chart for easy deployment of the very useful bot: [kubewatch](https://github.com/skippbox/kubewatch).

### CODE REVIEW

File: incubator/kubewatch/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: kubewatch
+version: 0.1.0
+appVersion: v0.0.2
```
Comment (unguiculus) [inline incubator/kubewatch/Chart.yaml]: v0.0.3

File: incubator/kubewatch/templates/configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
```
Comment (unguiculus) [inline incubator/kubewatch/templates/configmap.yaml:2]: Add standard labels.

File: incubator/kubewatch/templates/configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+metadata:
+  name: {{ template "kubewatch.fullname" . }}-config
+  namespace: {{ .Release.Namespace }}
```
Comment (unguiculus) [inline incubator/kubewatch/templates/configmap.yaml]: Why are you settings the namespace explicitly? Shouldn't be necessary.

File: incubator/kubewatch/templates/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+spec:
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      annotations:
+        checksum/config-map: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+      {{- if .Values.podAnnotations }}
+{{ toYaml .Values.podAnnotations | indent 8 }}
+      {{- end }}
+      labels:
```
Comment (unguiculus) [inline incubator/kubewatch/templates/deployment.yaml:20]: You can remove `heritage` and `chart` labels here.

File: incubator/kubewatch/templates/NOTES.txt [hunk 5]
```diff
@@ -0,0 +1,3 @@
+To verify that kubewatch has started, run:
+
+  kubectl --namespace={{ .Release.Namespace }} get pods -l "app={{ template "kubewatch.fullname" . }}"
```
Comment (unguiculus) [inline incubator/kubewatch/templates/NOTES.txt]: `-l "app={{ template "kubewatch.name" . }},release={{ .Release.Name }}`

File: incubator/kubewatch/README.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+`resources` | pod resource requests & limits | `{}`
+`slack.channel` | slack channel to notify | `""`
+`slack.token` | slack API token | `""`
+`tolerations` | List of node taints to tolerate (requires Kubernetes >= 1.6) | `[]`
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```console
+$ helm install incubator/kubewatch --name my-release \
+  --set=image.tag=v0.0.2,resources.limits.cpu=200m
```
Comment (unguiculus) [inline incubator/kubewatch/README.md]: I wouldn't use the image as an example.

File: incubator/kubewatch/Chart.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+description: Kubewatch notifies your slack rooms when changes to your cluster occur
+keywords:
+- kubernetes
+- slack
+maintainers:
+- name: Casey Kuhlman
```
Comment (unguiculus) [inline incubator/kubewatch/Chart.yaml]: Use Github username.

**Review by unguiculus — CHANGES_REQUESTED** [review]
Thanks for the PR.

File: incubator/kubewatch/templates/configmap.yaml (hunk 3 above)
Comment (compleatang) [reply incubator/kubewatch/templates/configmap.yaml]: oops.

File: incubator/kubewatch/templates/configmap.yaml (hunk 2 above)
Comment (compleatang) [reply incubator/kubewatch/templates/configmap.yaml:2]: done

File: incubator/kubewatch/templates/deployment.yaml (hunk 4 above)
Comment (compleatang) [reply incubator/kubewatch/templates/deployment.yaml:20]: done

**Comment by compleatang** [comment]
@unguiculus sorry for those challenges (should have refactored this chart based on your valuable feedback on my other submissions). Everything should be fixed up now.

**Comment by compleatang** [comment]
I've iterated the chart number but still the test is failing with:

```
Error please increment the new chart version to be greater than the existing version of 
```

I'm not sure how to overcome this? Are there any tips?

**Comment by unguiculus** [comment]
There's a problem with the CI script. It's being fixed.

**Review by unguiculus — APPROVED** [review]

**Comment by compleatang** [comment]
Thanks!

**Pull request merged**